Gemfile.lock

```diff
@@ -44,35 +44,35 @@ GEM
     loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    method_source (0.9.2)
+    method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
+    minitest (5.14.0)
     money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.8)
+    rack (2.2.2)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    railties (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
     rake (13.0.1)
     thor (1.0.1)
     thread_safe (0.3.6)
-    tzinfo (1.2.5)
+    tzinfo (1.2.6)
       thread_safe (~> 0.1)
-    zeitwerk (2.2.2)
+    zeitwerk (2.3.0)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.2 → 2.3.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.2.2...v2.3.0).

These are the individual commits:
- (ad26cb0) [allow eager loading after reloading](https://github.com/fxn/zeitwerk/commit/ad26cb0521cfb88fd8fee687dca7eb10e4ccf1bc)
- (0146d4c) [test Ruby 2.7 on CI](https://github.com/fxn/zeitwerk/commit/0146d4c3be3087a887cdfbbce7c697a0386c27f0)
- (af53354) [Merge pull request #103 from tricknotes/ruby-2.7](https://github.com/fxn/zeitwerk/commit/af53354543b943292eed1e29e6927e3771679905)
- (763e3dc) [Micro optimization: `[0...-3] = ''` is faster than `slice!(-…](https://github.com/fxn/zeitwerk/commit/763e3dc0d1d4cecded327e77f7c8f49d0a383c13)
- (527d548) [Merge pull request #105 from Shopify/micro-optim-slice](https://github.com/fxn/zeitwerk/commit/527d5489c9158ee596077d90491f0846a895868c)
- (2f6b32f) [annotates a type [skip ci]](https://github.com/fxn/zeitwerk/commit/2f6b32fd04fc11f9b853e4be7bece23d5f794847)
- (21e4341) [typo [skip ci]](https://github.com/fxn/zeitwerk/commit/21e434139c322b5bfba4bf9b733a27773987c216)
- (32c1cdb) [adds support for collapsing directories](https://github.com/fxn/zeitwerk/commit/32c1cdbca82c2314a951187f1e4719efb20b7557)
- (806795d) [version 2.3.0](https://github.com/fxn/zeitwerk/commit/806795d302840a7e96612b88ff45f231ea4318b0)


</details>